### PR TITLE
PIP-1338-increase-multipart-chunksize

### PR DESCRIPTION
**Pull request by paul-sud**

### CODE REVIEW

File: accession/accession.py [hunk 1]
```diff
@@ -305,10 +308,27 @@ def _upload_file_locally(self, encode_file: EncodeFile, gs_file: GSFile) -> None
...
+        s3.upload_fileobj(gcs_blob, bucket, key, Config=transfer_config)
         self.logger.info("Finished uploading file %s", filename)
 
+    def _calculate_multipart_chunksize(self, file_size_bytes: int) -> int:
+        """
+        Calculates the `multipart_chunksize` to use for `boto3` `TransferConfig` to
+        ensure that the file can be uploaded successfully without reaching the 100000
+        part limit. The default values are the same as the defaults for `TransferConfig`
+        """
+        multipart_chunksize = BOTO3_DEFAULT_MULTIPART_CHUNKSIZE * (
```
Comment (leepc12) [inline accession/accession.py:325]: If file size is exactly `BOTO3_DEFAULT_MULTIPART_CHUNKSIZE*BOTO3_MULTIPART_MAX_PARTS` then return value is `2 * BOTO3_DEFAULT_MULTIPART_CHUNKSIZE` even though `BOTO3_DEFAULT_MULTIPART_CHUNKSIZE` works fine with such case.

**Review by leepc12 — APPROVED** [review]
I left one comment about the default chunk size. Other than that LGTM.

File: accession/accession.py (hunk 1 above)
Comment (paul-sud) [reply accession/accession.py:325]: true, adjusted the formula to handle this

Comment (leepc12) [reply accession/accession.py:325]: Yup, looks good.

**Pull request merged**